dbus-daemon is in /usr/lib/ 

Not suitable for upstream

--- a/tests/meson.build	2019-12-04 10:49:09.404834167 -0800
+++ b/tests/meson.build	2019-12-04 10:49:29.858444080 -0800
@@ -49,7 +49,7 @@
 endforeach
 
 python3 = find_program('python3', required: false)
-dbus_daemon = find_program('dbus-daemon', required: false)
+dbus_daemon = find_program('/usr/lib/dbus-daemon', required: false)
 
 if python3.found() and dbus_daemon.found()
   test_dconf_py = find_program('test-dconf.py')
